docs/source/developer/interface/environment-variables.md

```diff
@@ -1,15 +1,33 @@
 # Environment Variables
 
-You can alter Interface's functionality using these environment variables.
+You can alter Interface's functionality using the following environment variables.
 
-Uses the source files for the QML and other resource at runtime instead of the compiled files.
+---
+
+Use the source files for QML and other resources at runtime instead of the compiled files.
 
     HIFI_USE_SOURCE_TREE_RESOURCES=1
 
-Enables the OpenVR plugin to be used with an Oculus HMD. (Useful for debugging the OpenVR plugin if you have an Oculus HMD but not a Vive.)
+---
+
+Enable the OpenVR plugin to be used with an Oculus HMD. (Useful for debugging the OpenVR plugin if you have an Oculus HMD but not a Vive.)
```

Comment:
```suggestion
Enable the OpenVR plugin to be used with an Oculus HMD. (Useful for debugging the OpenVR plugin if you have an Oculus HMD.)
```
There is many more SteamVR HMDs other than the Vive nowadays.

Reply:
This is a developer debugging aid: the Oculus controllers don't work properly with this setting enabled, for example, It just forces the iopenvr.dll plugin to be used instead of oculus.dll plugin. ... So I'd say no text change needed.

Reply:
Did you try that out? The Oculus controllers should work perfectly fine via OpenVR.

Reply:
Yes, I don't have a Vive but have used this setting for debugging the Vive plugin. The Oculus controllers move your hands OK but the button and trigger mappings aren't handled.